jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -242,10 +242,10 @@ ignition_collection_jobs =
         'ignition_msgs-ci-ign-msgs6-windows7-amd64',
         'ignition_msgs6-install-pkg-bionic-amd64',
         'ignition_msgs6-install_bottle-homebrew-amd64',
-        'ignition_physics-ci-ign-physics3-bionic-amd64',
-        'ignition_physics-ci-ign-physics3-homebrew-amd64',
-        'ignition_physics3-install-pkg-bionic-amd64',
-        'ignition_physics3-install_bottle-homebrew-amd64',
+        'ignition_physics-ci-ign-physics4-bionic-amd64',
+        'ignition_physics-ci-ign-physics4-homebrew-amd64',
```

Comment:
the `ign-physics4` branch doesn't exist yet; I suggest `main` instead of `ign-physics4`

* https://build.osrfoundation.org/job/ignition_physics-ci-main-bionic-amd64/

I recommend testing this dsl before merging to confirm it works

Reply:
Ah true, I updated the files, but I'm not sure which dsl to call to create the views.

Reply:
Ok, re-ran:

https://build.osrfoundation.org/job/_dsl_ignition_collection/308/

Here's the view:

https://build.osrfoundation.org/view/ign-edifice/

We're missing the install jobs but I think that's expected.